src/styles/blocks/calendar.scss

```diff
@@ -0,0 +1,46 @@
+
+.calendar {
+  position: relative;
+  top: 50%;
+  left: 50%;
+  transform: translate(-50%, -50%);
+
+
+  display: flex;
+  flex-wrap: wrap;
+  gap: $dayBoxGap;
+  width: getCalendarWidth($dayBoxSize, $calendarPadding, $dayBoxGap);
+  padding: 10px;
+
+  &__day {
+    display: flex;
+    justify-content: center;
+    align-items: center;
+    width:$dayBoxSize;
+    height: $dayBoxSize;
+
+    border: 1px solid #000;
+    background-color: #eee;
+
+    transition: 0.5s ease-in-out;
+    cursor: pointer;
+
+    &:hover {
+      background-color: pink;
+      transform: translate(0, -20px);
+    }
+
+    @for $date from 1 through 31 {
+      &:nth-child(#{$date})::before {
+        content: "#{$date}";
+      }
+    }
+
+    @each $day, $margin in $days {
+      .calendar--start-day--#{$day} &:first-child {
+        margin-left: $dayBoxSize * $margin + $margin;
+      }
+    }
```

Comment:
Let's move it higher to calendar styles because it's a modifier for the calendar.

Reply:
It's not working  in that way